modular_splurt/code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -262,3 +262,13 @@
 
 /obj/item/clothing/head/hooded/corpus/jp //sec
 	icon_state = "corpus_jp"
+
+/obj/item/clothing/suit/invisijacket
+	name = "invisifiber jacket"
+	desc = "A jacket made of transparent fibers, often used with reinforcement kits."
+	icon = 'modular_splurt/icons/obj/clothing/suits.dmi'
+	mob_overlay_icon = 'modular_splurt/icons/mob/clothing/suit.dmi'
+	// No overlay, because they're invisible!
+	icon_state = "jacket_transparent"
+	// I feel REALLY bad about doing this, but the default overlay is not transparent (null does not work either).
+	item_state = "jacket_purple"
```

Comment:
you could just make modular items_right/lefthand.dmi files with an empty icon state for the jacket and set its righthand and lefthand variables to those files and change the item_state accordingly

Reply:
as in adding left hand and right hand item files to store the transparent icon in, unless you found a way to make it transparent without having to add icons